volumes.c

```diff
@@ -112,39 +115,55 @@ static int pv_volumes_mount_volume(struct pantavisor *pv, struct pv_volume *v)
 	struct pv_state *s = pv->state;
 	char path[PATH_MAX], base[128], mntpoint[521];
 	char *fstype;
+	char *umount_cmd = NULL;
+	char *handlercut = NULL;
+	char *handler = NULL;
+	char *name = NULL;
+	const char *partname = NULL;
+	struct stat buf;
+	int wstatus;
+	char *command;
 
 	sprintf(base, "%s/disks", pv_config_get_storage_mntpoint());
 
+	handlercut = strchr(v->name, ':');
```

Comment:
I still think the dm: thingy would be better in json format in the state json. That way we could get rid of this code here, do the job in the parser, put it in the volume struct and take here whatever is in v

Reply:
yeah i hear you but lets try like this for now.... we will regret i  am sure :)